### PR TITLE
Fix freeRebirth flag

### PR DESCRIPTION
**Pull request by jobywalker**
Coffee is assuming the negation (!) applies to the AND of the two tests
should be just !user.flags.freeRebirth = true.  Fixes 4 test errors.

UserID: 7941e218-7b6e-42b7-989d-8b040c65aa17

### CODE REVIEW

**Comment by jobywalker** [comment]
This should also fix:  https://github.com/HabitRPG/habitrpg/issues/3350

"Can't buy Rebirth Orb"

**Comment by deilann** [comment]
Actually, that's super old and unrelated. That was pre-level cap removal.

The flag has nothing to do with purchasing the orb.

**Comment by jobywalker** [comment]
Ah ok.

File: script/index.coffee [hunk 1]
```diff
@@ -1155,7 +1155,7 @@ api.wrap = (user, main=true) ->
           dialog: i18n.t('messageFoundQuest', {questText: content.quests.vice1.text(req.language)}, req.language)
       if !user.flags.rebirthEnabled and (user.stats.lvl >= 50 or user.achievements.ultimateGear or user.achievements.beastMaster)
         user.flags.rebirthEnabled = true
-      if !user.flags.freeRebirth = true and user.stats.lvl >= 100
+      if user.stats.lvl >= 100 and !user.flags.freeRebirth = true
```
Comment (lefnire) [inline script/index.coffee:1158]: dunno if there's an additional bug here or it was intended, but `and !user.flags.freeRebirth = true` assigns.. should be `and !user.flags.freeRebirth == true` ? (or even just `and !user.flags.freeRebirth`?

Comment (jobywalker) [inline script/index.coffee:1158]: Yup completely missed that. Will submit another patch.

Comment (deilann) [inline script/index.coffee:1158]: Totally overlooked that. Sorry Tyler.

Comment (lefnire) [inline script/index.coffee:1158]: aint no thang, the bug was there before anyway
